api/app/Modules/Accounts/Http/api.php

```diff
@@ -22,6 +22,7 @@
         Route::put('/tracks', [Controllers\LibraryController::class, 'saveTracks'])
             ->middleware(ClearResponseCache::withTags(CacheTags::SAVED_TRACKS));
 
-        Route::delete('/tracks', [Controllers\LibraryController::class, 'removeSavedTracks']);
+        Route::delete('/tracks', [Controllers\LibraryController::class, 'removeSavedTracks'])
+            ->middleware(ClearResponseCache::withTags(CacheTags::SAVED_TRACKS));
```

Comment:
How come we are doing this again, i thought this was done already as part of a different PR?

Reply:
I forgot to add the middleware to clear the response cache on the `DELETE` endpoint here in the other PR.